types/gun/tslint.json

```diff
@@ -0,0 +1 @@
+{ "extends": "dtslint/dt.json", "rules": { "no-unnecessary-generics": false } }
```

Comment:
What is the reason for this being turned off? Generally we prefer that rules not be disabled unless it causes undue burden.

Reply:
You can see Gun constructor will return ChainReference<DataType> & GunSEA, if disable this rule, developers will have to write Gun() as ChainReference<DataType> & GunSEA.